### PR TITLE
dark mode animation

### PR DESCRIPTION
**Pull request by clarkmcadoo**
https://user-images.githubusercontent.com/26727669/172471685-4b6acf80-569b-4426-aacd-c3d63fd98535.mov

### CODE REVIEW

File: packages/gatsby-theme-newrelic/src/components/DarkModeToggle.js [hunk 1]
```diff
@@ -56,11 +56,37 @@ const DarkModeToggle = ({ className, size, onClick }) => {
       `}
     >
       <Icon
-        name={darkMode.value ? 'nr-dark-mode' : 'nr-light-mode'}
+        name="nr-light-mode"
```
Comment (caylahamann) [inline packages/gatsby-theme-newrelic/src/components/DarkModeToggle.js:59]: should we change the name of this icon to something like `nr-dark-mode-toggle` or vice versa?

**Review by caylahamann — COMMENTED** [review]
Looking at the preview, the icon is transposed if you start on dark mode so when you click it doesn't change, and also I'm not seeing the actual animation but I'm really not sure why that is

File: packages/gatsby-theme-newrelic/src/components/DarkModeToggle.js (hunk 1 above)
Comment (clarkmcadoo) [reply packages/gatsby-theme-newrelic/src/components/DarkModeToggle.js:59]: we should, its not defined in this repo though i don't think? at least I can't find it

Comment (clarkmcadoo) [reply packages/gatsby-theme-newrelic/src/components/DarkModeToggle.js]: lol it is, i fixed

**Pull request merged**